x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/rules_stats_collector.ts

```diff
@@ -65,141 +73,153 @@ const getRulesStatsQuery = (): SearchRequest => ({
   query: {
     match_all: {},
   },
+  runtime_mappings: getIdentifierRuntimeMapping(),
   aggs: {
-    rules: {
+    accounts: {
       terms: {
-        field: 'rule.id',
+        field: 'asset_identifier',
```

Comment:
mega nit, but I think its useful to add comments for runtime fields, since we create them from functions, the actual field names are not displayed in the query's `runtime_mappings` itself.

```
// generates the 'asset_identifier' field
runtime_mappings: getIdentifierRuntimeMapping(),
  aggs: {
    accounts: {
      terms: {
        // this field is runtime generated  
        field: 'asset_identifier',
```

Reply:
done:)